string location_name
geometry_msgs/Pose location_pose
---
string result_text
